_posts/2024-05-28-edb-and-kasten.markdown

```diff
@@ -0,0 +1,295 @@
+---
+layout: post
+title: Protect your postgres EDB database with Kasten
```

Comment:
Suggestion: Protect Postgres with EDB and Kasten
Please capitalize Postgres, regardless.

Reply:
done